src/PolyhedralGeometry/LinearProgram.jl

```diff
@@ -96,45 +96,11 @@ feasible_region(lp::LinearProgram) = lp.feasible_region
 ###############################################################################
 ###############################################################################
 
-"""
-    minimal_vertex(LP::LinearProgram)
-
-Return either a point of the feasible region of `LP` which minimizes the objective
-function of `LP`, or `nothing` if no such point exists.
-
-# Examples
-The following example constructs a linear program over the three dimensional cube.
-Although the linear program is given using the `:max` convention, one may still call
-`minimal_vertex`.
-```jldoctest
-julia> C=cube(3)
-A polyhedron in ambient dimension 3
 
-julia> LP=LinearProgram(C,[1,2,-3])
-The linear program
-   max{c⋅x + k | x ∈ P}
-where P is a Polyhedron and
-   c=Polymake.Rational[1 2 -3]
-   k=0
-
-julia> minimal_vertex(LP)
-pm::Vector<pm::Rational>
--1 -1 1
-```
 """
-function minimal_vertex(lp::LinearProgram)
-   mv = lp.polymake_lp.MINIMAL_VERTEX
-   if mv != nothing
-      return dehomogenize(mv)
-   else
-      return nothing
-   end
-end
+    optimal_vertex(LP::LinearProgram)
```

Comment:
Not to nitpick, but have you considered the term `extremal_nitpick`? AFAIK "extremal = minimal or maximal" is a very common convention in math. not so sure about "optimal"...

Reply:
That is how these functions are called in standard optimization software.  So, this choice is superior to "extremal".